src/utils/oidcConfig.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable sonarjs/no-nested-template-literals */
+import config from "../config";
+
+const port = window.location?.port || '';
+const baseUrl = `${window.location.protocol}//${window.location.hostname}${port ? `:${  port}` : ''}`;
+
+const commonOidcConfig = {
+  silent_redirect_uri: `${baseUrl}/silent-renew/`,
+  authority: config.openIdAuthority,
+  redirect_uri: `${baseUrl}/callback`,
+  post_logout_redirect_uri: `${baseUrl}/callback/logout`,
+}
+
+// profiili config
+export const userOidcConfig = {
+  client_id: config.openIdClientId,
+  scope: 'openid profile email',
+  ...commonOidcConfig,
+}
+
+export const adminOidcConfig = {
+  client_id: 'kerrokantasi-ui-staff-dev',
```

Comment:
I think should be able to give different client ids for different environment 🤔

Reply:
This has also been simplified as we decided not to implement two types of login paths.